tests/Silverback.Integration.Tests.E2E/Kafka/BasicTests.cs

```diff
@@ -1084,5 +1086,75 @@ public async Task Inbound_FromMultiplePartitionsWithLimitedParallelism_Concurren
 
             receivedMessages.Should().HaveCount(12);
         }
+
+        [Fact]
+        public void Outbound_BrokerEvents_CallbackCalled()
```

Comment:
Not everything has to be under `BasicTests`. We could think about a dedicated fixture for the event handlers?